gm4_midnight_menaces/data/sandy_husks/loot_tables/husk.json

```
@@ -0,0 +1,53 @@
+{
+  "pools": [
+    {
+      "rolls": 1,
+      "entries": [
+        {
+          "type": "item",
+          "name": "minecraft:rotten_flesh",
+          "weight": 1,
+          "functions": [
+            {
+              "function": "set_count",
+              "count": {
+                "min": 0,
+                "max": 2,
```

```suggestion
                "max": 2
```
There's a trailing comma here.